widgets/open-tickets/src/toolbar.php

```diff
@@ -57,16 +57,34 @@
 if ($preferences['toolbar_buttons']) {
     if (!isset($preferences['opened_tickets']) || $preferences['opened_tickets'] == 0) {
         if ($preferences['action_open_hosts']) {
-            $toolbar .= "<label id='buttontoolbar_4' style='font-size: 13px; font-weight: bold; cursor:pointer;' for='host-ticket'>Host <input type='image' title='" . _("Host: Open ticket") . "' alt='" . _("Host: Open ticket") . "' src='" . $centreon->optGen['oreon_web_path'] . "/modules/centreon-open-tickets/images/open-ticket.svg' name='host-ticket' style='border: none; width: 24px; height: 24px; vertical-align: middle;'/> </label> | ";
+            $toolbar .= "<label id='buttontoolbar_4' style='font-size: 13px; font-weight: bold; cursor:pointer;' " .
+            "for='host-ticket'>Host <input type='image' title='" .
+            _("Host: Open ticket") . "' alt='" . _("Host: Open ticket") . "' src='" .
+            $centreon->optGen['oreon_web_path'] .
```

Comment:
lines 63 and 64 should be concatenated. for a better readability, I'm used to add the "src" attrib from previous line too